beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -96,6 +96,24 @@ func setExecutionData(ctx context.Context, blk interfaces.SignedBeaconBlock, loc
 		// Compare payload values between local and builder. Default to the local value if it is higher.
 		localValueGwei := primitives.WeiToGwei(local.Bid)
 		builderValueGwei := primitives.WeiToGwei(bid.Value())
+		// Use local block if min bid is not attained
+		if builderValueGwei < primitives.Gwei(params.BeaconConfig().MinBuilderBid) {
+			log.WithFields(logrus.Fields{
+				"localGweiValue":   localValueGwei,
```

Comment:
Instead of local value, we should log `minBuilderBid` value here. Local value is not useful